docs/lifecycle.rst

```diff
@@ -0,0 +1,31 @@
+API Versions and Life Cycle
```

Comment:
This section is generally clear, but I think we can smoothen the language a bit. If you don't mind, I'd proofread this and add a commit with suggestions, which you can either accept or pick from. Ok for you?

Reply:
Go nuts, commit your changes, and let me know which id to cherry-pick

Reply:
See https://github.com/desec-io/desec-stack/pull/140/commits/d5b82b5e986382548ee87acf7ab634aa4c0c14be